src/App/redirects/UnsubscribeRedirect.tsx

```diff
@@ -0,0 +1,32 @@
+import {
+    generatePath,
+    useNavigate,
+    useParams,
+} from 'react-router-dom';
+
+import routes from '#routes';
+
+interface UnsubscribeParams {
+    subscriptionId: string | undefined;
+    token: string | undefined;
+    [key: string]: string | undefined;
```

Comment:
Do we need this?

Reply:
We need to pass Index signature for useParams.